docs/sources/storage/_index.md

```diff
@@ -11,85 +11,110 @@ is then compressed and stored in chunks in object stores such as S3 or GCS, or
+GCS is a hosted object store offered by Google. It is a good candidate for a managed object store, especially when you're already running on GCP, and is production safe.
 
-### S3
+#### Amazon Simple Storage Storage (S3)
 
 S3 is AWS's hosted object store. It is a good candidate for a managed object store, especially when you're already running on AWS, and is production safe.
 
-### Azure Blob Storage
+#### Azure Blob Storage
 
 Blob Storage is Microsoft Azure's hosted object store. It is a good candidate for a managed object store, especially when you're already running on Azure, and is production safe.
 You can authenticate Blob Storage access by using a storage account name and key or by using a Service Principal.
 
-### IBM Cloud Object Storage (COS)
+#### IBM Cloud Object Storage (COS)
+
 [COS](https://www.ibm.com/cloud/object-storage) is IBM Cloud hosted object store. It is a good candidate for a managed object store, especially when you're already running on IBM Cloud, and is production safe.
 
-### Notable Mentions
+#### Baidu Object Storage (BOS)
 
-You may use any substitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
+[BOS](https://intl.cloud.baidu.com/product/bos.html) is the Baidu CLoud hosted object storage.
 
-## Implementations - Index
+#### Alibaba Object Storage Service (OSS)
 
-### Single-Store
+[OSS](https://www.alibabacloud.com/product/object-storage-service) is the Alibaba Cloud hosted object storage.
 
-Single-Store refers to using object storage as the storage medium for both Loki's index as well as its data ("chunks"). There are two supported modes:
+#### Other notable mentions
 
-#### tsdb (recommended)
+You may use any substitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
 
-Starting in Loki v2.8, the [TSDB index store]({{< relref "../operations/storage/tsdb" >}}) improves query performance, reduces TCO and has the same feature parity as "boltdb-shipper".
+### Cassandra (deprecated)
 
-#### BoltDB (deprecated)
+Cassandra is a popular database and one of Loki's possible chunk stores and is production safe.
 
-Also known as "boltdb-shipper" during development (and is still the schema `store` name). The single store configurations for Loki utilize the chunk store for both chunks and the index, requiring just one store to run Loki.
+> **Note:** This storage type for chunks is deprecated and may be removed in future major versions of Loki.
 
-Performance is comparable to a dedicated index type while providing a much less expensive and less complicated deployment.
+## Index storage
```

Comment:
I think it would make sense to add one or two sentences here to explain that if you're using boltdb or TSDB you do not need a separate index storage. Otherwise it looks odd that everything under this heading is marked deprecated.